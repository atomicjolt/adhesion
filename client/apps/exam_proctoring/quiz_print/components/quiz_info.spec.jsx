import React            from 'react';
import TestUtils        from 'react-addons-test-utils';
import QuizInfo         from './quiz_info';
import Stub             from '../../../../specs_support/stub';

describe('Quiz Info', () => {
  let result;
  let props;

  beforeEach(() => {
    props = {
      title: 'Testacular',
    };
    result = TestUtils.renderIntoDocument(<Stub><QuizInfo {...props} /></Stub>);
  });

  it('renders the title', () => {
    const element = TestUtils.scryRenderedDOMComponentsWithTag(result, 'div')[0];
    expect(element.textContent).toContain('Testacular');
  });
});
