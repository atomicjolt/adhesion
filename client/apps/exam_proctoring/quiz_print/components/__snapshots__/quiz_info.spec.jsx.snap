// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Info matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <quizInfo
    title="Testacular"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": "Testacular",
    },
    "ref": null,
    "rendered": "Testacular",
    "type": "h2",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Testacular",
      },
      "ref": null,
      "rendered": "Testacular",
      "type": "h2",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
